client/signup/steps/plans/style.scss

```diff
@@ -1,6 +1,11 @@
 .plans-step {
 	margin: 0 auto;
 	max-width: 700px;
+
+	&.is-wide-layout {
```

Comment:
With redesigned plans components. Why: because it doesn't fit and `/plans` page uses it too. I also need to fix `.step-wrapper` because it has `max-width: 960px` only.